mono/mini/debugger-agent.c

```diff
@@ -7266,6 +7266,14 @@ event_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 				return err;
 			}
 
+			DebuggerTlsData *tls = (DebuggerTlsData *)mono_g_hash_table_lookup (thread_to_tls, THREAD_TO_INTERNAL(step_thread));
+			
+			if (tls->terminated) { 
```

Comment:
Can `tls` be null here?

Reply:
Yes, you are right, I changed how to get and the inserted the assert like is done in other places.